TRENZ.Docs.API.Test/TomlAuthzPermissionTableProviderTest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using TRENZ.Docs.API.Services;
+
+namespace TRENZ.Docs.API.Test;
+
+[TestClass]
+[TestCategory("Services")]
+public class TomlAuthzPermissionTableProviderTest
+{
+    [TestMethod]
+    public async Task TestGetPermissionTablesAsync()
+    {
+        var sources = TestHelper.ProvideFiles("./Data");
+        var logger = TestHelper.GetLogger<TomlAuthzPermissionTableProvider>();
+        var provider = new TomlAuthzPermissionTableProvider(sources, logger);
+
+        var tables = await provider.GetPermissionTablesAsync().ToListAsync();
+
+        Assert.AreEqual(3, tables.Count);
```

Comment:
this reads `TRENZ.Docs.API.Test/Data/.authz`? If so, shouldn't that be two tables? (if not, add a comment)